tooling/nargo_cli/tests/execution_success/tuple_inputs/src/main.nr

```diff
@@ -22,11 +22,7 @@ fn main(pair : (Field, Field), x: [(u8, u8, u8); 2], struct_pair: (Foo, Bar)) ->
     assert(struct_pair.0.bar.inner == [100, 101, 102]);
     assert(struct_pair.1.inner == [103, 104, 105]);
 
-    let (u, v) = if pair.0 as u32 < 1 {
-        (pair.0, pair.0 + 1)
-    } else {
-        (pair.0 + 1, pair.0)
-    };
+    let (u, v) = if pair.0 as u32 < 1 { (pair.0, pair.0 + 1) } else { (pair.0 + 1, pair.0) };
```

Comment:
Perhaps we should log the aggressiveness of ifs on one line as an issue too. I'm not sure what a better metric for them would be though since this one still fits on one line but reads better on multiple (imo).